javascripts/controllers/homeCtrl.js

```diff
@@ -1,3 +1,9 @@
-myApp.controller("homeCtrl", function($scope) {
-  
+myApp.controller("homeCtrl", function($scope, requestService) {
+  $scope.fetch = () => {
+    requestService.get($scope.url)
+  };
+});
+
+myApp.service("requestService", function($http) {
+  this.get = (url) => $http.get(url);
 });
```

Comment:
Given you've implemented the `homeCtrl` in [your app.js](https://github.com/nss-evening-cohort-04/foundations-fetch/pull/7/commits/812ede9acc42f86410d119be75c75478e91859b1#diff-c36fc7bc67d10b918d9b0a6563a7511bR5) and [not importing it into you index.html](https://github.com/nss-evening-cohort-04/foundations-fetch/pull/7/files#diff-eacf331f0ffc35d4b482f1d15a887d3bR29), feel free to delete this. :smile: